stdlib/public/Cxx/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ add_swift_target_library(swiftCxx STATIC NO_LINK_NAME IS_STDLIB IS_SWIFT_ONLY
 
     SWIFT_COMPILE_FLAGS ${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS} ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
     -cxx-interoperability-mode=default
-    -enable-experimental-feature Span
```

Comment:
I didn't do this change because I was thinking that an older compiler could encounter this usage of Span and complaint.

Reply:
Nevermind, looks like it should still be printed. Brought it back here: https://github.com/swiftlang/swift/pull/79419